mappings/net/minecraft/entity/player/PlayerInventory.mapping

```diff
@@ -14,6 +28,9 @@ CLASS net/minecraft/class_1661 net/minecraft/entity/player/PlayerInventory
 		ARG 1 shouldRemove
 		ARG 2 maxCount
 		ARG 3 craftingInventory
+	METHOD method_32338 offer (Lnet/minecraft/class_1799;Z)V
+		ARG 1 stack
+		ARG 2 transmitUpdate
```

Comment:
Maybe `syncToClient`? I forgot the convention for such parameters.

Reply:
`sendToClient` might work here, since I see it has a `instanceof ServerPlayerEntity` as well.

I wouldn't use "sync" because that implies a two-way communication. This is just sending from the server to the client.

Reply:
```suggestion
		ARG 2 notifiesClient
```
Otherwise it feels like some sort of block update etc

Reply:
@Sollace how you do think of this name suggestion? Imo this pr is good besides this name.

Reply:
```suggestion
		ARG 2 notifiesClient
```

Reply:
@liach I'll allow it. ;3